src/main/resources/application.yml

```diff
@@ -55,7 +55,7 @@ codefreak:
     fileSystem:
       # The path where all file collections will be stored in sub-directories
       # Please make sure that the user running the server has read+write permissions on this directory
-      collectionStoragePath: "/var/lib/codefreak"
+      collectionStoragePath: "${user.home}/.codefreak/${codefreak.instanceId}/collections"
```

Comment:
Do we need some sort of file migration for people that already had their files in `/var/lib/codefreak`? Or at least mention it somewhere?

Reply:
No because there was no actual release, yet.